### DISCUSSION TITLE
Zigbee2MQTT Errore

### DISCUSSION THREAD

**Discussion by Moxser** [indomus/forum#663] [ZigBee, Z-Wave, Thread, Matter]
Ciao A tutti.

Spero che qualcuno mi possa aiutare.
Dopo l'aggiornamento da Zigbee2MQTT 1.20.0-1 a 1.21.0-4 non mi funziona + la chiavetta CC2530 munita di un amplificatore CC2592. 
Ho provato a ripristinare uno snapshot prima dell'aggiornamento ma niente da fare.

Questa è al mia configurazione
![image](https://user-images.githubusercontent.com/72693930/131552327-7893915f-68e6-4e72-8481-086585676615.png)

Questi sono gli errori che sono presenti nel registro:

[19:55:14] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.20.0 start /app
> node index.js
Zigbee2MQTT:info  2021-08-31 19:55:18: Logging to console and directory: '/share/zigbee2mqtt/log/2021-08-31.19-55-16' filename: log.txt
Zigbee2MQTT:info  2021-08-31 19:55:19: Starting Zigbee2MQTT version 1.20.0 (commit #1.20.0)
Zigbee2MQTT:info  2021-08-31 19:55:19: Starting zigbee-herdsman (0.13.117)
Zigbee2MQTT:error 2021-08-31 19:55:38: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2021-08-31 19:55:38: Failed to start zigbee
Zigbee2MQTT:error 2021-08-31 19:55:38: Check https://www.zigbee2mqtt.io/information/FAQ.html#help-zigbee2mqtt-fails-to-start for possible solutions
Zigbee2MQTT:error 2021-08-31 19:55:38: Exiting...
Zigbee2MQTT:error 2021-08-31 19:55:39: Error: Failed to connect to the adapter (Error: SRSP - SYS - ping after 6000ms)
    at ZStackAdapter.<anonymous> (/app/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:93:31)
    at Generator.throw (<anonymous>)
    at rejected (/app/node_modules/zigbee-herdsman/dist/adapter/z-stack/adapter/zStackAdapter.js:25:65)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! zigbee2mqtt@1.20.0 start: `node index.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the zigbee2mqtt@1.20.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-08-31T17_55_39_067Z-debug.log

Se qualcuno è cosi gentile da aiutarmi gli sarei veramente grato.
Grazie mille a tutti

**Comment by snakuzzo** [comment]
E' un bug della 1.21...devi tornare alla 1.20

**Reply by Moxser** [reply]
Ho ripristinato la 1.20 con uno snapshot ma senza risultati e con gli stesso errori. Hai qualche suggerimento o un modo migliore per tornare alla versione 1.20?

**Reply by Moxser** [reply]
Alla fine ho dovuto togliere e reinserire la chiavetta per farla leggere correttamente dal sistema.

**Reply by snakuzzo** [reply]
Ho fatto anche io un test ed in effetti, anche rirpristinando la 1.20, ho tolto ed inserito in altra porta USB ed è andato

**Reply by Moxser** [reply]
Grazie mille per l'aiuto come sempre!!

**Reply by Moxser** [reply]
Ciao a tutti per caso qualcuno ha avuto modo di aggiornare Zigbee2MQTT alla versione 1.21.1_1?
Io con la versione 1.21.0-4 ci ho riprovato ancora un paio di volte ma ho sempre dovuto ripristinare la versione 1.20 e poi togliere e reinserire la chiavetta. Ci ho rinunciato. 
Sapete se con l'ultimo aggiornamento è stato risolto il bug?

**Reply by essegi65** [reply]
Da quello che ho visto ieri su github ancora è in lavorazione una soluzione. anche io sto aspettando ho lo stesso problema.
Dove posso trovare la versione 1.20 da installare su hassio con supervisor?
Grazie

**Reply by snakuzzo** [reply]
Seguite [questa issue](https://github.com/Koenkk/zigbee2mqtt/issues/8282#issuecomment-925585919).
Dagli ultimi test che abbiamo fatto con koenkk 2 giorni fa, è uscito fuori che dovrebbe essere un problema legato alla nuova funzionalità di backup introdotta di recente con i coordinator basati su CC2530.

**Comment by filpao** [comment]
Come ho scritto in altro post, ho lo stesso problema, che rimane anche facendo il downgrade alla 1.20.
Il problema è che io non posso togliere la chiavetta e rimetterla, perchè la mia antenna è standalone attaccata ad un esp8266.
Ho anche provato ESP-Easy al posto di ESP-Link, ma senza risultati (ho seguito la guida https://www.zigbee2mqtt.io/information/connecting_cc2530.html)

Vi posto le mie config, :
Su ESP8266 con ESP-Easy abbiamo tutto di default, a parte Wifi e Devices, che è come segue:

> 

> Communication - Serial Server
> 
> Name: ZIGBEE2MQTT
> Enabled: checked
> 
> Sensor
> Serial Port: HW Serial0: GPIO-3 (D9) ← TX / GPIO-1 (D10) → RX
> TCP Port: 1775
> Baud Rate: 152000
> Serial Config: 8N1
> Event processing: Generic
> RX Receive Timeout (mSec): 0
> 
> Data Acquisition
> Single event with all values: uncecked


Su Zigbee2Mqtt abbiamo

> data_path: /config/zigbee2mqtt
> external_converters: []
> devices: devices.yaml
> groups: groups.yaml
> homeassistant: true
> permit_join: false
> mqtt:
>   base_topic: zigbee2mqtt
>   server: mqtt://localhost:1883
>   user: IlMioUtente
>   password: LaMiaPassword
> serial:
>   port: tcp://192.168.1.135:1775
> advanced:
>   log_level: warn
>   pan_id: 6754
>   channel: 11
>   network_key:
>     - 1
>     - 3
>     - 5
>     - 7
>     - 9
>     - 11
>     - 13
>     - 15
>     - 0
>     - 2
>     - 4
>     - 6
>     - 8
>     - 10
>     - 12
>     - 13
>   availability_blocklist: []
>   availability_passlist: []
> device_options: {}
> blocklist: []
> passlist: []
> queue: {}
> frontend:
>   port: 8099
> experimental: {}
> socat:
>   enabled: false
>   master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
>   slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
>   options: '-d -d'
>   log: false

Grazie mille.
Filippo

**Reply by Moxser** [reply]
Filippo l'unica cosa che puoi fare, non potendo togliere e reinserire la chiavetta, è aspettare che nei prossimi giorni risolvano il problema. 
Considerando che è un errore riscontrato da molti sicuramente troveranno una soluzione. Eventualmente se ci sono novità scriviamo qui.

**Reply by snakuzzo** [reply]
1. Vi pregherei di non aprire ulteriori thread sulla stessa discussione. Fate reply sotto l'ultimo messaggio.
2. È inutile che riportate i vostri problemi. Il problema è noto e si sa pure la causa. Leggete [QUI](https://github.com/indomus/forum/discussions/663#discussioncomment-1384353)
